### DISCUSSION TITLE
No Copilot suggestions or ouput of any kind in VS Code

### DISCUSSION THREAD

**Discussion by Iron27** [community/community#40655] [Copilot Conversations] [answered]
I just signed up and paid for the annual subscription to Copilot.  I installed the extension in VS Code.

I went through the initial setup in Github and VS Code and everything looks correct: in VS Code the Copilot icon is bottom right without notice, my Github account icon is bottom right also without notice, and under "Manage Trusted Extensions" even confirms that Github Copilot "last used this account 10 seconds ago".

In VS Code: Copilot is enabled.  Inline suggestions are enabled.  I have tried both varieties of Github's Copilot setting: "Suggestions matching public code" -- Allow and Block.  Neither work.  I have restarted VS Code multiple times.  Nothing changes.

I've typed all variety of suggested triggers from the startup guide, eg:  

`function calculateDaysBetweenDates(begin, end)`, 

`// find all images without alternate text
// and give them a red border
function process() {`

`// Express server on port 3000`

I get no code suggestions of any kind, no output in the GitHub Copilot Console, and no errors.  The extension is fully setup and all indicators / icons say it's working, but it does nothing - good, bad, or otherwise.

I'm excited to use this extension and any help is appreciated.

**Comment by mohamedzhioua** [comment]
@Iron27  you can use Blackbox (vscode extension) it's similar to capilote

**Comment by shang-lin** [comment, marked as answer]
I had the same issue in VS Code on Mac. I tried deleting the cache as in one of the answers at [https://stackoverflow.com/questions/67698176/error-loading-webview-error-could-not-register-service-workers-typeerror-fai]. It didn't seem to work when I restarted VS Code, but then I tried hitting CTRL-Enter and got a tab of Copilot suggestions. After that, Copilot just started working.

**Reply by Iron27** [reply]
CTRL + Enter does the trick.  Copilot still fails to automatically recommend code, but I honestly don't mind hitting CTRL + Enter every time I want suggestions anyway. 

Thanks!

**Comment by rickCoin** [comment]
Same here.
The copilt only work when I open the Completions Panel.
When I close the Completions Panel, there would no any in-line suggestions. Event if I tried the Keyboard Shortcuts for editor.action.inlineSuggest.trigger, it still shows nothing.

Have anyone solved it before?
